AutoFollowIT.testAutoFollowManyIndices failure

**Build scan**:
https://gradle-enterprise.elastic.co/s/vgarx75evvhp4

**Repro line**:
```
./gradlew ':x-pack:plugin:ccr:internalClusterTest' --tests "org.elasticsearch.xpack.ccr.AutoFollowIT.testAutoFollowManyIndices" \
  -Dtests.seed=BBFCB0557D17ACF7 \
  -Dtests.security.manager=true \
  -Dtests.locale=sk \
  -Dtests.timezone=America/Menominee \
  -Dcompiler.java=14 \
  -Druntime.java=11
```

**Reproduces locally?**:
No

**Applicable branches**:
Master

**Failure history**:
Rare, only twice in 31 days. It is not clear whether the failure is a due to an intermittent build issue or the actual test so I leaving the test unmuted. 

Another instance: https://gradle-enterprise.elastic.co/s/xqbrmt6iaglg4

**Failure excerpt**:
```
java.lang.AssertionError: 
Expected: a collection with size <13>
     but: collection size was <10>
	at __randomizedtesting.SeedInfo.seed([BBFCB0557D17ACF7:93CE9F2814860980]:0)
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
	at org.junit.Assert.assertThat(Assert.java:956)
	at org.junit.Assert.assertThat(Assert.java:923)
	at org.elasticsearch.xpack.ccr.AutoFollowIT.lambda$testAutoFollowManyIndices$7(AutoFollowIT.java:199)
	at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:892)
	at org.elasticsearch.xpack.ccr.AutoFollowIT.assertLongBusy(AutoFollowIT.java:562)
	at org.elasticsearch.xpack.ccr.AutoFollowIT.testAutoFollowManyIndices(AutoFollowIT.java:191)
```

--- a/x-pack/plugin/ccr/src/internalClusterTest/java/org/elasticsearch/xpack/ccr/AutoFollowIT.java
+++ b/x-pack/plugin/ccr/src/internalClusterTest/java/org/elasticsearch/xpack/ccr/AutoFollowIT.java
@@ -559,7 +559,7 @@ public class AutoFollowIT extends CcrIntegTestCase {
 
     private void assertLongBusy(CheckedRunnable<Exception> codeBlock) throws Exception {
         try {
-            assertBusy(codeBlock, 60L, TimeUnit.SECONDS);
+            assertBusy(codeBlock, 120L, TimeUnit.SECONDS);
         } catch (AssertionError ae) {
             AutoFollowStats autoFollowStats = null;
             try {

